public/locales/es/account.json

```diff
@@ -67,6 +67,7 @@
     "field_required": "Este campo es obligatorio",
     "monthly": "Mensual",
     "monthly_subscription": "Suscripción mensual",
+    "no_pricing_available": "No hay opciones de precios disponibles en este momento.",
```

Comment:
We have decided to create a separate PR for Spanish keys (ott-webapp channel). Did you agree on this one with Jose?

Reply:
Ah, didn't know what's been decided there. So I just revert these changes and open different PR only for this string, then check with Jose?

Reply:
Yeah, let's try it this way. Thanks!